plugins/agent_types/AcquiaLiftAgent.inc

```diff
@@ -1024,97 +901,6 @@ class AcquiaLiftAgent extends PersonalizeAgentBase implements PersonalizeAgentGo
     }
     return $items;
   }
-
-  /**
-   * Implements AcquiaLiftAgentInterface::syncFixedTargeting().
-   */
-  public function syncFixedTargeting($option_sets) {
-    $items = $this->getFixedTargetingSyncOperations($option_sets);
-    $this->queueItems($items);
-  }
-
-  /**
-   * Returns the operations needed to sync goals to Acquia Lift.
-   *
-   * @param $option_sets
-   *   An array representing the option sets whose targeting rules need to be sync'd.
-   * @return array
-   *    An array of items representing API calls to be made to Acquia Lift.
-   */
-  public function getFixedTargetingSyncOperations($option_sets) {
```

Comment:
This function is still called from the batch process.
Fatal error: Call to undefined method AcquiaLiftAgent::getFixedTargetingSyncOperations() in /Users/lisa.backer/Documents/workspace/lift/acquia_lift/acquia_lift.batch.inc on line 196